### PR TITLE
Fix formatting in G-lettered articles

### PR DESCRIPTION
**Pull request by TPGPL**
Includes:
- fixed formatting for Guides folder
- fixed formatting for Game Modifiers folder
- osu!standard -> osu!

### CODE REVIEW

File: wiki/Guides/Recording_osu!/fr.md [hunk 1]
```diff
@@ -1,22 +1,20 @@
 # Faire des enregistrements d'osu!
 
-[Vous voulez utiliser Virtualdub au lieu de Sony Vegas ? Cliquez ici !](https://osu.ppy.sh/forum/viewtopic.php?p=252802#p252802)
+[Vous voulez utiliser Virtualdub au lieu de Sony Vegas ? Cliquez ici !](https://osu.ppy.sh/community/forums/topics/18112?start=252802#p252802)
```
Comment (cl8n) [inline wiki/Guides/Recording_osu!/fr.md]: remove anchor

File: wiki/Guides/Recording_osu!/en.md [hunk 2]
```diff
@@ -1,22 +1,20 @@
 # How to record videos of osu!
 
-[Want to use Virtualdub instead of Sony Vegas? Click here!](https://osu.ppy.sh/forum/viewtopic.php?p=252802#p252802)
+[Want to use Virtualdub instead of Sony Vegas? Click here!](https://osu.ppy.sh/community/forums/topics/18112?start=252802#p252802)
```
Comment (cl8n) [inline wiki/Guides/Recording_osu!/en.md]: remove anchor

File: wiki/Guides/Beginner's_Tutorial/de.md [hunk 3]
```diff
@@ -1,48 +1,50 @@
+# Tutorial for beginners (Deutsch)
```
[the remark this answers is not in the document]
Comment (TPGPL) [reply wiki/Guides/Beginner's_Tutorial/de.md]: If there is no title in the translation, we add the English one and native language name in brackets. At least that's what we were doing in the past. Gonna keep it and remove all of them in another PR if thats a problem

**Comment by TPGPL** [comment]
applied everything

File: wiki/Guides/Recording_osu!/en.md (hunk 2 above)
Comment (cl8n) [reply wiki/Guides/Recording_osu!/en.md]: I meant only the part after #, cuz this is supposed to link to the specific post https://osu.ppy.sh/community/forums/topics/18112?start=252802 (or https://osu.ppy.sh/community/forums/posts/252802)

**Pull request merged**